pkgs/os-specific/linux/nfacct/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl, pkg-config, libnetfilter_acct, libmnl }:
+
+stdenv.mkDerivation rec {
+  pname = "nfacct";
+  version = "1.0.2";
+
+  src = fetchurl {
+    url = "https://netfilter.org/projects/${pname}/files/${pname}-${version}.tar.bz2";
```

```suggestion
    url = "https://netfilter.org/projects/nfacct/files/nfacct-${version}.tar.bz2";
```

This makes it easier to browse the URL in the browser while reviewing.